packages/bigquery/sodasql/dialects/bigquery_dialect.py

```diff
@@ -35,12 +35,19 @@ class BigQueryDialect(Dialect):
     def __init__(self, parser: Parser):
         super().__init__(BIGQUERY)
         if parser:
-            self.account_info_dict = self.__parse_json_credential('account_info_json', parser)
             self.dataset_name = parser.get_str_required('dataset')
             default_auth_scopes = ['https://www.googleapis.com/auth/bigquery',
                                    'https://www.googleapis.com/auth/cloud-platform',
                                    'https://www.googleapis.com/auth/drive']
             self.auth_scopes = parser.get_list_optional('auth_scopes', default_auth_scopes)
+            self.__context_auth = parser.get_str_optional('use_context_auth', None)
+            if self.__context_auth:
+                self.account_info_dict = None
+                self.project_id = parser.get_str_required('project_id')
+                logger.info("You are using context auth, json provided are ignore.")
```

Comment:
Minor edit, please change the log message to :

`Using context auth, account_info_json will be ignored.`